shared/src/tests/mockdata.ts

```diff
@@ -207,22 +207,19 @@ export const fullAccessToAllContentMap = {
     "group-private-content": {
         post: {
             view: true,
-            create: true,
```

Comment:
Remember to also remove this from all the seeding docs in the API (search for `"create"`)